node/src/main/scala/coop/rchain/node/metrics.scala

```diff
@@ -0,0 +1,25 @@
+package coop.rchain.node
+
+import kamon.prometheus._
+import kamon._
+import scala.util.control.NonFatal
+
+final case class Metrics(port: Int = 9095) {
+  val reporter = new PrometheusReporter()
+
+  Kamon.addReporter(reporter)
+
+  /*
+   * Small interface to demonstrate the External Node API.
+   */
+
+  def start(): Unit =
```

Comment:
uuu already not liking this :) i would prefer not adding yet another method with eager effect. especially since I just fixed `connect` . @kirkwood can we work on that together so it adhere to changes that I'm doing? I will be more then happy to do some pair programming on it

Reply:
We will have to :)